UnitTests/BraintreePayPalNativeCheckoutTests/BTPayPalNativeTokenizationClient_Tests.swift

```diff
@@ -74,4 +73,33 @@ class BTPayPalNativeTokenizationClient_Tests: XCTestCase {
             }
         }
     }
+  
+    func testTokenizationRequestFailureResponseForNilReturnUrl() throws {
+        let mockClient = try XCTUnwrap(MockAPIClient(authorization: "development_client_key"))
+        let mockCorrelationId = "mock-corrID"
+
+        /// Same response data as the successful response, but the nonce is absent
+        let responseData = try XCTUnwrap("""
+        {
+            "paypalAccounts": [{
+                "consumed": 0,
+                "description": "PayPal",
+                "details": {
+                    "correlationId": "\(mockCorrelationId)",
+                },
```

Comment:
This test should include the `nonce` and still pass

```suggestion
                },
                "nonce": "\(mockNonce)",
```

Reply:
done